src/files_api.js

```diff
@@ -2,55 +2,79 @@ import * as pathmod from 'path-browserify';
 
 function extract_favorites_from_document(doc) {
   let favorites = [];
-  for(let element of doc.querySelectorAll('li[title="Files"] a[title]').values()) {
+
+  for (let element of doc
+    .querySelectorAll('li[title="Files"] a[title]')
+    .values()) {
+    let { title, href } = element;
+
+    // Interactive regex explorer:
+    // https://regex101.com/r/7sfoEw/1
+    let extractPath = new RegExp(
+      'pun/(sys|dev)/(?:ood-)?dashboard/files/fs(?<path>.*)',
```

Comment:
Is this because you run `/pun/dev/ood-dashboard` instead of just `dashboard`? Seems like a recipe for future confusion, regex's being hard enough to being with.

Reply:
`dashboard\/files\/fs(?<path>.*)` is much more simple, you're right!